docs/dev-guide/iframe.md

```diff
@@ -3,46 +3,85 @@ id: dev-guide-iframe
 title: IFrame API
 ---
 
-You can use the Jitsi Meet API to embed Jitsi Meet in to your application. You are also welcome to use it for embedding the globally distributed and highly available deployment on meet.jit.si itself. The only thing we ask for in that case is that you please DO NOT remove the jitsi.org logo from the top left corner.
+The IFrame API enables you to embed Jitsi Meet functionality into your meeting application so you can experience the full functionality of the ______.
 
-## Installation
+Additional functionality includes embedding the globally distributed and highly available deployment on meet.jit.si itself. 
 
-To embed Jitsi Meet in your application you need to add the Jitsi Meet API library:
+Note: Jitsi ask that you not remove the jitsi.org logo from the top left corner of the implementation.
```

Comment:
We can drop this now, since it's enforced.

Reply:
Removed